examples/plugins/Android.bp

```diff
@@ -0,0 +1,15 @@
+cc_library {
+    name: "libsample_pcm_plugin",
```

Comment:
Please rename to libtinyalsav2_example_plugin_pcm

Reply:
done